app/riot/src/server/framework-preset-riot.js

```diff
@@ -15,5 +15,12 @@ export function webpack(config) {
         },
       ],
     },
+    resolve: {
+      ...config.resolve,
+      alias: {
+        ...config.resolve.alias,
+        'riot-compiler': 'riot-compiler/dist/es6.compiler',
```

Comment:
I am not sure what happened (maybe env changes) but riot-compiler is now resolved to riot-compiler/lib/compiler which is a node version and riot app is failing.
This is change is a workaround, but must be fixed in this PR

Reply:
oops, I don't know what I can do about it.

Reply:
@ndelangen, this issue was not addressed but introduced in this PR...